### ISSUE TITLE
use current model

### ISSUE THREAD

**Issue by m3m3-chan** [harry-7/speech-emotion-recognition#5]
Hello, 
can we use the current model without training?

**Comment by hkveeranki** [comment]
Yes. You can. But my code doesn't provide direct one step process to it. after you create the model. instead of training them you can load the model. using inbuilt method `load_weights` of keras. 

This is a very good feature to have. I will add this functionality soon

**Comment by hkveeranki** [comment]
This is now supported. After you create a model, use  `.restore_model()` with the *saved path* of the model to assign weights and then you can use `.predict()` to predict the labels or `.evaluate()`. 

Fixed in 538a6de
